src/include/distributed/connection_management.h

```diff
@@ -28,8 +28,8 @@
 /* used for libpq commands that get an error buffer. Postgres docs recommend 256. */
 #define ERROR_BUFFER_SIZE 256
 
-/* application name used for internal connections in Citus */
-#define CITUS_APPLICATION_NAME "citus"
+/* application name prefix used for internal connections in Citus */
+#define CITUS_APPLICATION_NAME_PREFIX "citus gpid="
```

Comment:
needs rebase as we have changed it to `citus_internal`. I think, I'd stick to `citus_internal`

Reply:
I'm making it "citus_internal gpid=" then